Neos.Flow/Scripts/flow.php

```diff
@@ -39,7 +39,16 @@
     array_shift($argv);
     require(__DIR__ . '/migrate.php');
 } else {
-    $composerAutoloader = require(__DIR__ . '/../../../Libraries/autoload.php');
+    $autoloadFilePath = dirname(__DIR__, 3) . '/Libraries/autoload.php';
```

Comment:
Ah wait, you changed the variable name, but below we have `$bootstrap = new \Neos\Flow\Core\Bootstrap($context, $composerAutoloader);`

Reply:
Doh, I completely overlooked that.. And I would suggest not to rename the variable here